templates/index

```diff
@@ -202,6 +202,7 @@ cisco_ios_dir.textfsm,  .*, cisco_ios, dir
 cisco_nxos_show_l2rib_internal_permanently-frozen-list.textfsm, .*, cisco_nxos, sh[[ow]] l2ri[[b]] i[[nternal]] pe[[rmanently-frozen-list]]
 cisco_nxos_show_configuration_session_summary.textfsm, .*, cisco_nxos, sh[[ow]] configu[[ration]] s[[ession]] su[[mmary]]
 cisco_nxos_show_interface_transceiver_details.textfsm, .*, cisco_nxos, sh[[ow]] int[[erface]] tra[[nsceiver]] de[[tails]]
+cisco_nxos_show_interface_transceiver.textfsm, .*, cisco_nxos, sh[[ow]] int[[erface]] tra[[nsceiver]]
```

Comment:
Should be ordered in length and is a bit too high up in the ordering. I think it's only a few lines off.

If you have **tox** installed, you should be able to run **tox** from the root directly and it'll build the necessary environment and run the tests.